src/scss/elements/_default.scss

```diff
@@ -1,11 +1,12 @@
 /// Base styles for a teaser
 /// sets the font and basic display properties
 @mixin oTeaserBase {
-	@include oTypographyProgressiveFont(sans, m);
+	@include oTypographySans(-1);
+	@include oTypographyMargin($bottom: 4);
 	position: relative;
-	margin-bottom: 15px;
-	text-rendering: optimizeLegibility;
 	width: 100%;
+	text-rendering: optimizeLegibility;
```

Comment:
It is in `o-normalise` if you use the CSS reset part of it, but can't be sure everyone is using that at the moment.